homeassistant/components/glances/__init__.py

```diff
@@ -49,7 +47,6 @@
             vol.Optional(CONF_USERNAME): cv.string,
             vol.Optional(CONF_PASSWORD): cv.string,
             vol.Optional(CONF_SSL, default=False): cv.boolean,
-            vol.Optional(CONF_VERIFY_SSL, default=True): cv.boolean,
```

Comment:
Please don't change the config schema for the YAML config. That's deprecated and we should just ignore the ssl option if it's irrelevant now.

Reply:
👍🏻